web/app/js/components/TopRoutes.jsx

```diff
@@ -62,24 +61,23 @@ class TopRoutes extends React.Component {
       PrefixedLink: PropTypes.func.isRequired,
     }).isRequired,
     classes: PropTypes.shape({}).isRequired,
-    query: topRoutesQueryPropType,
+    query: topRoutesQueryPropType.isRequired,
+    setQuery: PropTypes.func.isRequired,
   }
-  static defaultProps = {
-    query: {
+
+  constructor(props) {
+    super(props);
+    this.api = this.props.api;
+
+    const query = _merge({
```

Comment:
We need to send by default all values to empty strings to avoid issues with mandatory prop `value` for `SelectInput` component